### DISCUSSION TITLE
Interoperability failure

### DISCUSSION THREAD

**Discussion by Peaceandmaths** [iterate-ch/cyberduck#15171] [Q&A]
I uploaded 12 large zip files to Cyberduck and successfully expanded 11 of them. When I try to unzip the problematic one I receive the following error : 

<img width="884" alt="Interoperability failure" src="https://github.com/iterate-ch/cyberduck/assets/117741432/f2827f10-5674-46ee-ab56-4d077fd2bfd4">

I checked that the file is indeed seen by the server : I can get its location, type and stats. 
![image](https://github.com/iterate-ch/cyberduck/assets/117741432/02594cfc-cf20-458e-b0a6-59db4002d70e)

After some chatting with ChatGPT, I tried to solve the multi-part archive problem, but wasn't successful : 

<img width="865" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/117741432/892b669e-0bad-4d05-9dbf-769b4782f854">

I could try to reload the file, but it's very heavy and might take days again, so if anyone could give me a hint on how to solve this, would be great :)

**Comment by AliveDevil** [comment]
Please check data integrity by comparing an MD5 checksum against your source and uploaded file:
- On your source open PowerShell and run `Get-FileHash -Algorithm MD5 <File Name>`
- On your remote server run `md5sum <File Name>`.

Did you verify that you could open the Internal_TR2 archive locally?
